lib/siilar/client.rb

```diff
@@ -75,6 +74,10 @@ def base_options
         }
       }
 
+      if requests_timeout
+        options.merge!(timeout: requests_timeout)
```

Comment:
Mmh I would say that we let it like that, for both. What do you think about it ?